machine/dell/dellemc_s4100_c2338/rootconf/sysroot-etc/inittab

```diff
@@ -0,0 +1,16 @@
+# /etc/inittab for onie
```

Comment:
why is this file included?  I haven't seen that done before.

Reply:
it is needed for micro-usb console to work , which as it uart-1

Reply:
OK, I see.

At some point we may want to handle this in a more general way -- making a wholesale copy of inittab is a little dangerous.

Another approach would be to allow machines to *append* platform specific lines to the main inittab file at build time.  Somewhat like an include file mechanism.  Otherwise we run the risk of a machine missing out on some core ONIE updates to the inittab file.